### PR TITLE
LINKTA-309: CD/Configure firebase hosting

### PR DESCRIPTION
**Pull request by ChTiSh**
## Description
Configured Firebase hosting for waiting list page.
Integration with Next will need to be configured as well.
This is a breaking change, putting PR as draft first so everyone is aware.

Fixes # (issue) LINKTA-309

## Type of change

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 Documentation update
- [X] Other 🗒️ CD

## How Has This Been Tested?

Test will be performed after Nextjs integration

## Checklist:

- [X] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I have reviewed the [**Security Best Practices Document**](SECURITY_BEST_PRACTICES.md).
- [X] I have followed the [**Naming Conventions Guide**](NAMING_CONVENTIONS_GUIDE.md) for my changes.
- [X] I have ensured that my pull request title is descriptive.

### CODE REVIEW

File: public/404.html
Comment (dcpavel) [inline public/404.html:1]: Why is the error page within this PR? Also, the styling isn't consistent with the rest of the styling (not using tailwind). Is this part of firebase?

File: public/index.html
Comment (dcpavel) [inline public/index.html:1]: Same as the 404 page, styling is inconsistent with the rest of the waiting list site. Is this part of firebase?

**Review by dcpavel — DISMISSED** [review]
I'm a little confused if this is only the CD and configuring firebase hosting.

File: public/404.html
Comment (ChTiSh) [reply public/404.html:1]: Yeah, auto generated by firebase, just removed.

File: public/index.html
Comment (ChTiSh) [reply public/index.html:1]: Same as 404, auto-generated by firebase, also removed

**Review by kinskiwu — APPROVED** [review]

**Review by colinvandergraaf — APPROVED** [review]
This looks good.  Looking forward to testing this more thoroughly once we have it all integrated. BTW, I watched an earlier version of this video about  [ default settings in firebase configurations let to easily hackable apps](https://www.youtube.com/watch?v=TKyNPg7UIIc) because proper security rules aren't implemented by default.

**Review by zhiyi15 — APPROVED** [review]

**Pull request merged**